packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -19,8 +19,8 @@ export interface DailyReturnType {
   url: string;
   created_at: string;
   config: {
-    nbf: number;
-    exp: number;
+    nbf: number /* Timestamps expressed in seconds, not in milliseconds */;
+    exp: number /* Timestamps expressed in seconds, not in milliseconds */;
```

Comment:
So these comments actually appear on the IDE.

```suggestion
    /** Timestamps expressed in seconds, not in milliseconds */
    nbf: number;
    /** Timestamps expressed in seconds, not in milliseconds */
    exp: number;
```

![imagen](https://user-images.githubusercontent.com/3504472/156208397-eaa56902-c7b9-4f93-879d-ba37885ef684.png)